features/steps/partcad-cli/commands/version.py

```
@@ -38,6 +68,15 @@ def step_impl(context, substring):
     assert substring not in context.result.stdout
 
 
+@then('STDERR should not contain "{substring}" with path')
+def step_impl(context, substring):
```

Ruff (0.8.2)

Redefinition of unused `step_impl` from line 63

(F811)